connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnectorTest.java

```diff
@@ -151,7 +155,14 @@ public void testFindConsumerGroups() throws Exception {
         doReturn(groups).when(connector).listConsumerGroups();
         doReturn(true).when(connector).shouldReplicateByTopicFilter(anyString());
         doReturn(true).when(connector).shouldReplicateByGroupFilter(anyString());
-        doReturn(offsets).when(connector).listConsumerGroupOffsets(anyString());
+
+        Map<CoordinatorKey, KafkaFuture<Map<TopicPartition, OffsetAndMetadata>>> futures = new HashMap<>();
+        futures.put(CoordinatorKey.byGroupId("g1"), KafkaFuture.completedFuture(offsets));
+        futures.put(CoordinatorKey.byGroupId("g2"), KafkaFuture.completedFuture(offsets));
+        ListConsumerGroupOffsetsResult offsetsResult = new ListConsumerGroupOffsetsResult(futures);
+        offsetsResult = spy(offsetsResult);
```

Comment:
I think you can avoid needing to change the constructor visibility if you use mock(ListConsumerGroupOffsetsResult.class) instead.